Packs/HackerOne/Integrations/HackerOne/HackerOne.yml

```diff
@@ -128,15 +128,13 @@ script:
       required: false
       secret: false
     - default: false
-      defaultValue: '50'
```

Comment:
Why have we deleted this?

Reply:
Because the default values of page_size and page_number will also be passed when the user just wants to work with the `limit` argument.

Reply:
Good point, tnanks

Reply:
This make sense 👍 , need to mention in the description that the default is 50 (you are enforcing it in the code)